lib/move_loader.rb

```diff
@@ -0,0 +1,19 @@
+require 'web_human_player'
+
+class MoveLoader
+  def preload(player, move)
+    if web_human?(player) && valid_move?(move)
+      player.set_move(move.to_i)
+    end
+  end
+
+  private
+
+  def web_human?(player)
+    player.class == WebHumanPlayer
```

Comment:
In order to preload the WebHumanPlayer with the move selected from the gui, I need to do a cast (as only a WebHumanPlayer has the set_move method). 

This could be made polymorphic if all player types had a set_move method, which did nothing for the other player types, however I felt that may be a LSP violation, as you would expect the method to do something, but it wouldn't always do it.